src/pages/Home/CustomizeBobaModalBody.tsx

```diff
@@ -21,9 +44,39 @@ const BobaModalform = ({ milk, setMilk }: { milk: string; setMilk: (milk: string
         <Col xs={12} md={8}>
           Milk(Required)
           <Form.Group>
-            <CustomizeRadioButton milkList={milkList} milk={milk} setMilk={setMilk} />
+            {Object.keys(milkList).map((key) => {
+              const milkOption = milkList[key];
+              return (
+                <CustomizeRadioButton
+                  type={'radio'}
+                  id={key}
+                  key={key}
```

Comment:
I think these are duplicates
```
id={key}
key={key} // remove this ?
```

Reply:
nah , i think customzeradio button need key cuz its iterating using that tag otherwise it give me this 
![image](https://user-images.githubusercontent.com/103973779/228114273-b91a8cf3-74b4-4748-a9e9-168dfd66eec9.png)

Reply:
Yeh, then remove `id` which is html version of React `key`
We don't need both
https://legacy.reactjs.org/docs/lists-and-keys.html#keys-must-only-be-unique-among-siblings

Reply:
seem like i need both, key is for iteration but not able to get it on child component ,

Reply:
Try this, if this doesn't work. Just keep the working id then
https://bobbyhadz.com/blog/react-get-key-of-clicked-element

Reply:
i m going to keep it as it is then